app/src/main/java/com/sdp13epfl2021/projmag/database/UserDataFirebase.kt

```diff
@@ -108,4 +109,19 @@ class UserDataFirebase(
             )
         }
     }
+
+    override fun pushCv(
+        cv: CurriculumVitae,
+        onSuccess: () -> Unit,
+        onFailure: (java.lang.Exception) -> Unit
+    ) {
+        getUserDoc()?.let {
+            doc ->
+            doc.set(
+                hashMapOf("cv" to cv),
+                SetOptions.merge()
+            ).addOnSuccessListener { onSuccess }
+                .addOnFailureListener(onFailure)
```

Comment:
The onSuccess was never call with ()
```suggestion
            ).addOnSuccessListener { onSuccess() }
                .addOnFailureListener(onFailure)
```